dart/lib/src/hub.dart

```diff
@@ -275,13 +276,13 @@ class Hub {
   }
 
   /// Clones the Hub
-  Hub clone() {
+  FutureOr<Hub> clone() async {
```

Comment:
```suggestion
  Future<Hub> clone() async {
```

Only `ScopeCallback` should be FutureOr because the user can define either an async or sync method.
`clone` is always `async`.